src/app/modules/item/components/item-permissions/item-permissions.component.html

```diff
@@ -81,7 +81,7 @@
           Note that some subgroups or users of the groups may have higher permissions.
         </div>
 
-        <ng-container *ngIf="permissions.canGrantView !== 'none' && watchedGroup.currentUserCanGrantGroupAccess">
+        <ng-container *ngIf="permissions.canGrantView !== 'none' && watchedGroup.currentUserCanGrantGroupAccess && itemData?.item?.permissions">
```

Comment:
`item` does not seem to be optional in `itemData`

Reply:
Still the template optional chaining issue, TS detects it properly but angular templates don't :shrug: